### PR TITLE
updating readme file for badges

### PR DESCRIPTION
**Pull request by marcelniyi**
---

#### What does this PR do?

> update readme.

#### Description of Task to be completed?

> Adding a working badges not dummy data

#### How should this be manually tested?

> switch to  "chore-update-readMe-#175743399" branch and see badges from README.md file

### Any background context you want to provide?

> **N/A**

### What are the relevant pivotaltracker stories?

> **https://www.pivotaltracker.com/n/projects/2476677/stories/175743399**

### Screenshots (if appropriate)

> **N/A**

### Questions

> **N/A**

### CODE REVIEW

**Review by ConstantineMaliza — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Barefoot Nomad
 
-[![Build Status](https://travis-ci.com/atlp-rwanda/space-bn-frontend.svg?branch=develop)](https://travis-ci.com/atlp-rwanda/space-bn-frontend)
-<a href="https://codeclimate.com/github/atlp-rwanda/space-bn-frontend/maintainability"><img src="https://api.codeclimate.com/v1/badges/3e75b6bbdd43289b59ec/maintainability" /></a>
-<a href="https://codeclimate.com/github/atlp-rwanda/space-bn-frontend/test_coverage"><img src="https://api.codeclimate.com/v1/badges/3e75b6bbdd43289b59ec/test_coverage" /></a>
-[![Coverage Status](https://coveralls.io/repos/github/atlp-rwanda/space-bn-frontend/badge.svg?branch=develop)](https://coveralls.io/github/atlp-rwanda/space-bn-frontend?branch=develop)
```
Comment (SimonImanigirimpuhwe) [inline README.md:6]: we still also need coverage badge

**Review by SimonImanigirimpuhwe — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (SimonImanigirimpuhwe) [inline README.md:6]: We still also need coverage badge

**Review by SimonImanigirimpuhwe — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Anguandia) [reply README.md:6]: @SimonImanigirimpuhwe please confirm if your change request has been attended to and resolve

Comment (marcelniyi) [reply README.md:6]: the change is resolved

**Review by marcelniyi — COMMENTED** [review]
everything is fine now

**Review by ConstantineMaliza — APPROVED** [review]

**Review by SimonImanigirimpuhwe — APPROVED** [review]
Good work!

**Pull request merged**